jans-cedarling/cedarling/src/jwt/decoding_strategy/key_service.rs

```diff
@@ -12,12 +12,13 @@ pub use error::Error;
 use jsonwebtoken::jwk::JwkSet;
 use jsonwebtoken::DecodingKey;
 use openid_config::*;
-use reqwest::blocking::get;
+use reqwest::blocking::Client;
 use std::collections::HashMap;
 use std::sync::Arc;
 
 pub struct KeyService {
     idp_configs: HashMap<Box<str>, OpenIdConfig>, // <issuer (`iss`), OpenIdConfig>
+    http_client: Client,
 }
 
 #[allow(unused)]
```

Comment:
This silences a couple of unused variable warnings - `iss` in `for (iss, conf`, and `config` in `for (iss, config` in `fn get_key`.

Also it hides an unused Result in `self.update_jwks(iss)`